app/decorators/exercise_decorator.rb

```diff
@@ -3,9 +3,6 @@ class ExerciseDecorator < Draper::Decorator
 
   def status
     status = h.current_user.exercise_status_hash[object.id]
-    if (status.nil?)
-      status = "not started"
-    end
-    return status
+    status.present? ? status : status = "not started"
```

Comment:
This hash sums up the information of the UserExercises table. Since this table only stores started or finished exercises by the user, the hash will not have keys for the not started exercises. If this is the case, the status of the exercise will be "not started", meaning we don't have to add any special icon the the view (such as the finished or started icons)